elements/pfe-dropdown/src/pfe-dropdown-item.scss

```diff
@@ -0,0 +1,69 @@
+@import "../../pfe-sass/pfe-sass";
+@import "_shared-assets";
+
+$LOCAL: dropdown-item;
+
+$dropdown-item-variables: (
+  FontSize: #{pfe-var(font-size)},
+  FontWeight: 400,
+  LineHeight: #{pfe-var(line-height)},
+  Color: #151515,
+  Color--hover: #151515,
+  TextAlign: left,
+  BackgroundColor: transparent,
+  BackgroundColor--hover: #{pfe-color(surface--lighter)},
+  Border: none,
+);
+
+$variables: map-merge($variables, $dropdown-item-variables);
+
+:host {
+  @include pfe-print-variables($variables);
+  .pfe-dropdown-item {
+    &__container {
+      &:hover,
+      &:focus,
+      &:active {
+        color: #{pfe-local(Color--hover)};
+        background-color: #{pfe-local(BackgroundColor--hover)};
+      }
+    }
+  }
+}
+
+::slotted(*) {
+  display: block;
+  width: 100%;
+  padding: #{pfe-local(SpacingVertical)} #{pfe-local(SpacingHorizontal)};
+  font-size: #{pfe-local(FontSize)};
+  font-weight: #{pfe-local(FontWeight)};
+  line-height: #{pfe-local(LineHeight)};
+  color: #{pfe-local(Color)};
+  text-align: #{pfe-local(TextAlign)};
+  white-space: nowrap;
+  background-color: #{pfe-local(BackgroundColor)};
```

Comment:
@eyevana I just noticed most of the variables in this file are used on the slotted content, which means they are light DOM and could be overridden anyway like this:
```css
pfe-dropdown-item a, pfe-dropdown-item button {
    text-align: center;
}
```
@castastrophe is there any value in keeping the CSS variables?

Reply:
I think so.  You could add !important to them too to prevent light DOM from overriding.  It's nice to have a clear hook.